changelog.txt

```diff
@@ -1,3 +1,11 @@
+Version 3.0.3
+----------
+- Cut down logs (#1046)
+
+Version 3.0.2
```

Comment:
I have updated the changelog updates on 3.0.2 release branch.

Reply:
tag : [v3.0.2](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/releases/tag/v3.0.2)